a10_octavia/controller/worker/tasks/cert_tasks.py

```diff
@@ -63,143 +64,153 @@ class SSLKeyCreate(task.Task):
     """Task to create an SSL Key"""
     @axapi_client_decorator
     def execute(self, cert_data, vthunder):
-        try:
-            if self.axapi_client.file.ssl_key.exists(file=cert_data.key_filename):
-                self.axapi_client.file.ssl_key.update(file=cert_data.key_filename,
-                                                      cert=cert_data.key_content,
-                                                      size=len(cert_data.key_content),
-                                                      action="import")
-            else:
-                self.axapi_client.file.ssl_key.create(file=cert_data.key_filename,
-                                                      cert=cert_data.key_content,
-                                                      size=len(cert_data.key_content),
-                                                      action="import")
-        except Exception as e:
-            LOG.exception("Failed to create SSL key: %s", str(e))
-            raise e
+        if cert_data is not None:
```

Comment:
This shouldn't be needed. We've designed the flows so that cert_data will never be none if this task is exectued.